multichain-aggregator/multichain-aggregator-logic/src/repository/api_keys.rs

```diff
@@ -16,3 +19,16 @@ pub async fn find_by_key_and_chain_id(
 
     Ok(api_key)
 }
+
+pub async fn upsert_many<C>(db: &C, api_keys: Vec<ApiKey>) -> Result<(), DbErr>
```

Comment:
nit: is this function only for tests? 😮 

i would either move it to tests of add `cfg(test)` to it